websites/S/Stake/presence.ts

```diff
@@ -0,0 +1,80 @@
+const presence = new Presence({
+		clientId: "1030599970551771266",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const [showBal, showTimestamp, showCurrentGame] = await Promise.all([
+			presence.getSetting<boolean>("showBal"),
+			presence.getSetting<boolean>("showTimestamp"),
+			presence.getSetting<boolean>("showCurrentGame"),
+		]),
+		presenceData: PresenceData = {
+			largeImageKey: "logo",
+		};
```

Comment:
Since you use `window.location.pathname` and `.search` a lot, it may make sense to destructure it.
```suggestion
		},
		{ pathname, search } = window.location
```